src/styles/blocks/calendar.scss

```diff
@@ -1,30 +1,30 @@
 .calendar {
   display: flex;
-  padding: $ver-paddings $hor-paddings;
+  width: 7 * $cell-size + 6 * $gaps + 2 * $hor-paddings;
   margin: 0 auto;
+  padding: $ver-paddings $hor-paddings;
   gap: $gaps;
-  width: 7 * $cell-size + 6 * $gaps + 2 * $hor-paddings;
   flex-wrap: wrap;
 
   @each $weekDay, $numDay in $weekDays {
-    &--start-day_#{$weekDay} > :first-child {
+    &--start-day--#{$weekDay} > :first-child {
       margin-left: $numDay * ($cell-size + $gaps);
     }
   }
 
   @for $monthLength from 28 through 31 {
-    &--month-length_#{$monthLength} > :nth-child(n + #{$monthLength + 1}) {
+    &--month-length--#{$monthLength} > :nth-child(n + #{$monthLength + 1}) {
       display: none;
     }
   }
 
   &__day {
     @include square($cell-size);
     display: flex;
+    flex-shrink: 0;
```

Comment:
do u think u need shrink? cuz i cant see what it changes

Reply:
no changes, you right. Fixed